src/main/resources/static/js/answerSurvey.js

```diff
@@ -30,3 +34,52 @@ $(document).ready(() => {
         })
     });
 });
+
+function loadAnsweringDiv(question) {
+
+    var questionHTML = '<div class="input-group-prepend">\n' +
+        '           <span class="input-group-text">' + question.prompt + '</span>\n' +
+        '           </div>\n';
+
+    switch ($('#surveyTypeValue')[0].value) {
+        case "TEXT" :
+            questionHTML+='<textarea class="form-control" id="surveyTextArea" rows="3" name="answer"></textarea>';
+            break;
+        case "MC" :
+            var choices = question.choices;
+            questionHTML+='<div class="form-control">';
+            for(var i = 0; i < choices.length; i++) {
+                questionHTML+='<input type=\"radio\" id=\"'+choices[i]+'\" name=\"answer\" value=\"'+choices[i]+'\">';
+                questionHTML+='<label for="'+choices[i]+'">'+choices[i]+'</label>';
+            }
+            questionHTML+='<div>';
+            break;
+        case "NUMBER" :
+            questionHTML+='<div class="form-control">';
+            questionHTML+='<label for="cars">Choose a number:</label>';
+            questionHTML+='<select id="answer" name="answer">';
+            for( var i = question.min; i<=question.max; i++){
```

Comment:
Just considering the case where the range is something like 1-100, you could use something like this
`<input` type="number" id="answer" name="answer" min="question.min" max="question.max">`
rather than have a select with 100 options to scroll through and it should enforce the min and max requirements